charts/plane-enterprise/templates/workloads/pi-migrator.job.yaml

```
@@ -0,0 +1,50 @@
+          {{- toYaml .Values.extraEnv | nindent 10 }}
+        {{- end }}
+      restartPolicy: OnFailure
+      serviceAccount: {{ .Release.Name }}-srv-account
+      serviceAccountName: {{ .Release.Name }}-srv-account
+
+---
+{{- end }}
+
+
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (4 > 0)

(empty-lines)